app/src/main/java/org/simple/clinic/bp/history/BloodPressureHistoryScreenEffect.kt

```diff
@@ -1,10 +1,12 @@
 package org.simple.clinic.bp.history
```

Comment:
The commit message indicates that we are loading both the patient details as well as the list of BPs when this screen is created. However, that is not the case. What is actually happening is that this commit changes the screen so that the patient details are loaded in addition to the list of BPs which were already being done.

Can we reword this commit message? For ex: `Load patient details when the BP history screen is created`